__performance__/README.md

```diff
@@ -67,12 +67,14 @@ The following paths will be expected to be available:
 
 ### Setup
 
-From within the one-app directory start the one-app prod-sample:
+From within the one-app directory first start the one-app prod-sample:
 
 ```bash
 npm run start:prod-sample
 ```
 
+> `npn run start:prod-sample` creates the `prod-sample_one-app-at-test-network` network which is required for the performance tools to start when running against prod-sample.
```

Comment:
```suggestion
> `npm run start:prod-sample` creates the `prod-sample_one-app-at-test-network` network which is required for the performance tools to start when running against prod-sample.
```
I ran start:prod-sample first but it didnt create the docker network that the perf docker compose required. Might need to mention the command to create it manually if things fail.

Reply:
for now i am happy with the error provided by docker compose, its fairly explanatory. it takes a while for the prod-sample to finish setting up and actually start the network. maybe you jumped the gun as the one-app server start without that network

Reply:
As long as this is addressed later, Im good with this.